### PR TITLE
Update metrics doc for 1.0.0-m1 release

### PR DESCRIPTION
**Pull request by zhengyangyong**
update metrics docs for 1.0.0-m1 release

### CODE REVIEW

**Comment by zhengyangyong** [comment]
Merge https://github.com/apache/incubator-servicecomb-website/pull/34 first

**Comment by zhengyangyong** [comment]
Rebased

File: _docs/cn/quick-start-bmi.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ last_modified_at: 2017-09-04T10:01:43-04:00
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>io.servicecomb</groupId>
+        <groupId>org.apache.servicecomb</groupId>
         <artifactId>java-chassis-dependencies</artifactId>
-        <version>0.5.0</version>
+        <version>0.6.0-SNAPSHOT</version>
```
Comment (WillemJiang) [inline _docs/cn/quick-start-bmi.md]: Need to change the 1.0.0-m1

File: _docs/quick-start-bmi.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ Before we start, we need to add some common dependencies in the parent project o
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>io.servicecomb</groupId>
+        <groupId>org.apache.servicecomb</groupId>
         <artifactId>java-chassis-dependencies</artifactId>
-        <version>0.5.0</version>
+        <version>0.6.0-SNAPSHOT</version>
```
Comment (WillemJiang) [inline _docs/quick-start-bmi.md]: The version should be changed.

File: _docs/cn/quick-start-bmi.md (hunk 1 above)
Comment (zhengyangyong) [reply _docs/cn/quick-start-bmi.md]: Done

File: _docs/quick-start-bmi.md (hunk 2 above)
Comment (zhengyangyong) [reply _docs/quick-start-bmi.md]: Done

**Comment by zhengyangyong** [comment]
Fix these problem in https://github.com/apache/incubator-servicecomb-website/pull/34 an rebased this branch

File: _users/cn/setup-environment.md [hunk 3]
```diff
@@ -31,19 +31,19 @@ last_modified_at: 2017-09-03T10:01:43-04:00
    
    <div class="tab-content">
      <div id="windows" class="tab-pane active" markdown="1">
-   1. 下载[服务注册中心可执行文件压缩包](https://github.com/apache/incubator-servicecomb-service-center/releases/download/0.5.0/service-center-0.5.0-windows-amd64.zip)
+   1. 下载[服务注册中心可执行文件压缩包](http://www-eu.apache.org/dist/incubator/servicecomb/incubator-servicecomb-service-center/1.0.0-m1/apache-servicecomb-incubating-service-center-1.0.0-m1-windows-amd64.tar.gz)
```
Comment (WillemJiang) [inline _users/cn/setup-environment.md]: Please use  the below link http://apache.org/dyn/closer.cgi/incubator/servicecomb/incubator-servicecomb-service-center/1.0.0-m1/apache-servicecomb-incubating-service-center-1.0.0-m1-windows-amd64.tar.gz

Comment (zhengyangyong) [reply _users/cn/setup-environment.md]: Done

**Pull request merged**